app/src/main/kotlin/no/nav/tiltakspenger/vedtak/repository/behandling/kvp/KVPVilkårDbJson.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.tiltakspenger.vedtak.repository.behandling.kvp
+
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.kvp.KVPVilkår
+
+/**
+ * Har ansvar for å serialisere/deserialisere KvpVilkår til og fra json for lagring i database.
+ */
+internal data class KVPVilkårDbJson(
```

Comment:
Du dupliserer alle domene-klassene i repo-laget så ikke domene-klassene skal lagres direkte, og vi får muligheten til å endre det ene uten å endre det andre?

Reply:
Ja, akkurat det jeg ønsker :) Spesielt det at man kan stole på kodebasen, slik at man kan endre domenet, uten å være redd for at integrasjonene endres.